src/gui/advancedsettings.cpp

```diff
@@ -335,6 +337,8 @@ void AdvancedSettings::loadAdvancedSettings()
     spinBoxCheckingMemUsage.setValue(session->checkingMemUsage());
     spinBoxCheckingMemUsage.setSuffix(tr(" MiB"));
     addRow(CHECKING_MEM_USAGE, tr("Outstanding memory when checking torrents"), &spinBoxCheckingMemUsage);
+    item(CHECKING_MEM_USAGE, PROPERTY)->setToolTip(QString("<body><p>%1</p></body>")
```

Comment:
Can you try without `<body><p>` tags? such as `setToolTip(QString("%1"))`
If it is required, also try  `setToolTip(QString("<p>%1</p>"))`

Reply:
Without rich text it shows it in one line which is out of the question. Yesterday i started with just \<p> but there were some weird stuff in formatting and it was corrected when i added \<body>.
Today i tried a couple with and without body but they are the same. Does \<body> cause some overhead or something?

Reply:
Not sure. However if you intend to use rich text format, I think you should follow what qt designer uses: `<html><head/><body><p>text</p></body></html>`.

Reply:
Done.